data/helpers.d/mysql

```diff
@@ -90,3 +90,47 @@ ynh_mysql_create_user() {
 ynh_mysql_drop_user() {
     ynh_mysql_execute_as_root "DROP USER '${1}'@'localhost';"
 }
+
+# Create a database, an user and its password. Then store the password in the app's config
+#
+# User of database will be stored in db_user's variable.
+# Name of database will be stored in db_name's variable.
+# And password in db_pwd's variable.
+#
+# usage: ynh_mysql_generate_db user name
+# | arg: user - Owner of the database
+# | arg: name - Name of the database
+ynh_mysql_generate_db () {
+	db_pwd=$(ynh_string_random)	# Generate a random password
+	ynh_mysql_create_db "$2" "$1" "$db_pwd"	# Create the database
```

Comment:
Since this function is relatively high-level, wouldn't it make sense to automatically call `ynh_make_valid_dbid` to sanitize the db's name ? We could also store the db's name in the app settings as well.